thunder/benchmarks/targets.py

```diff
@@ -724,6 +725,35 @@ def backward_fn(result, forward_inputs, output_grads):
     return backward_fn, backward_setup
 
 
+# Thunder executor: RuntimeError: Advanced indexing currently only supports zero or one-dimensional integer tensors,
+# but found a tensor with dtype thunder.dtypes.int64 and 2 dimensions
+moe_executors = (torch_executor, torch_compile_executor, thunderfx_executor)
+moe_executors_ids = (
+    "torch",
+    "torch.compile",
+    "thunderfx",
+)
+
+
+@pytest.mark.parametrize(
+    "executor,",
+    moe_executors,
+    ids=moe_executors_ids,
+)
+@pytest.mark.parametrize(
+    "compute_type,",
+    (ComputeType.INFERENCE,),
+    ids=("inference",),
+)
+def test_MoE(benchmark, executor: Callable, compute_type: ComputeType):
```

Comment:
Let's rename the benchmark to include "DeepSeek" configuration and the project name the implementation is taken from.